Code/GraphMol/MolDraw2D/MolDraw2DSVG.cpp

```diff
@@ -456,4 +454,32 @@ void MolDraw2DSVG::outputClasses() {
   }
   d_os << "' ";
 }
+
+template <class t_obj> void MolDraw2DSVG::outputTagClasses(const t_obj *obj) const {
+  if (!d_activeClass.empty()) {
+    d_os << " " << d_activeClass;
+  }
+  if (obj->hasProp("_tagClass")) {
+    std::string value;
+    obj->getProp("_tagClass", value);
+    std::replace(value.begin(), value.end(), '\"', '_');
+    std::replace(value.begin(), value.end(), '\'', '_');
+    std::replace(value.begin(), value.end(), '.', '_');
+    d_os << " " << value;
+  }
+}
+
+template <class t_obj> void MolDraw2DSVG::outputMetaData(const t_obj *obj) const {
+  const std::regex quot_re("\"");
```

Comment:
At this point we are still kind of trying to support g++ v4.8, which means that using the standard C++ regex in code isn't really "allowed" (they still occasionally show up in tests, but should not be in the library itself). I suspect will lift this restriction in the 2021.09 release cycle and allow full use of C++11 and C++14 features, but in the meantime, if you really want to use a regex here, please look at the way they are handled in `MolFileParser.cpp`

In this particular case, I don't think the regex is actually adding anything, so it's probably better to switch back to just using `std::string` anyway.

Reply:
I've switched to using boost::replace_all which works just fine this really simple usage.

I just assumed that gcc 4.8 contained working regexp implementation based on dim memory of using it but it seems to be in truly "here be dragons"-territory, as the code compiles but does not do what you expect.

Reply:
yeah, the fact that code using std::regex compiles but doesn't actually work is really bothersome.